E8EB11FFFE254CF9.json

```diff
@@ -3,7 +3,7 @@
 		"gateway_ID": "E8EB11FFFE254CF9", 
 		"servers": [ 
 			{ 
-				"server_address": "router.eu.thethings.network", 
+				"server_address": "router.asia-se.thethings.network", 
```

Comment:
This is not a valid address. Check here for valid ones: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Reply:
@gonzalocasas 
Does this mean I need to change the frequency at gateway registration upon changing it here? currently Im using India 865-867Mhz?

Reply:
@gonzalocasas ,
Thanks for reply.
RAK831 is the board Im working on. @global_conf are you suggesting to change the frequency on radio0 and radio1?

Reply:
Yes, but check this master files for how to change that exactly: https://github.com/TheThingsNetwork/gateway-conf